authentication/understanding-identity-provider.adoc

```diff
@@ -78,3 +78,10 @@ include::modules/authentication-remove-kubeadmin.adoc[leveloffset=+1]
 include::modules/identity-provider-parameters.adoc[leveloffset=+1]
 
 include::modules/identity-provider-default-CR.adoc[leveloffset=+1]
+
+include::modules/identity-provider-provisioning-user-lookup-mapping.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/solutions/6006921[How to create user, identity and map user and identity in LDAP authentication for `mappingMethod` as `lookup` inside the OAuth manifest]
```

Comment:
I know you didn't do this, but the names of these solutions are very grammatically strange IMO. I had to read it multiple times to understand what "user, identity and map user and identity" meant.

Just wanted to note that. :slightly_smiling_face:

Reply:
lol yeah, unfortunately not a lot I can do about them though 😄